### ISSUE TITLE
Code review request

### ISSUE THREAD

**Issue by navv1234** [simulationcraft/simc#1842]
Originally reported on Google Code with ID 1843

```
Revision: r17264

Purpose of code changes on this branch:
Implementing better direct result tracking/output


The "Direct Results" breakdown should now include entries like:
hit
hit (blocked)
hit (crit-blocked)
glance
glance (blocked)

and so on. 

I've left the "Block Results" breakdown table in for this revision.  I'm not sure if
it's worth keeping or not.  It provides an at-a-glance reference of all blocked and
unblocked attacks, which may be useful for directly comparing to the average blocked
hit on a World of Logs parse, as I think they do a similar averaging.  But strictly
speaking, you could do that using the data under "Direct Results" with a weighted average.

In addition, removing the "Block Results" table would make the blocked_direct_results
and blocked_tick_results stat_results_t objects redundant, and they could be removed.
 I've already eliminated all references to those structures outside of this table.

So basically, I'm asking what the other tanks think about keeping/axing this additional
table.

```

Reported by `theckhd` on 2013-08-14 18:00:11
